js/tests.js

```diff
@@ -1,5 +1,165 @@
-QUnit.test("hello test", function(assert) {
-  assert.strictEqual(1 + 1, 2, "One plus one is two");
+QUnit.test("SuperHuman basics", function(assert) {
```

Comment:
I would suggest a description like "sets all the initial properties correctly", just to make it more clear the kinds of things you're testing.